src/components/presentation-menu.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme'
+import getMuiTheme from 'material-ui/styles/getMuiTheme'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+
+import ViewModeToggleButton from './view-mode-toggle-button'
+
+class PresentationMenu extends React.Component {
```

Comment:
We should rename this to `PresentationModeToolbar` for now. (It's really just a single button, so not really a toolbar.) The name of the file can be `presentation-mode-toolbar.jsx`.